test/e2e/features/minimal.feature

```diff
@@ -10,9 +10,17 @@ Feature: Minimal user story
         And starting CRC with default bundle succeeds
         Then checking that CRC is running
 
+        @podman-preset
         Examples:
             | preset-value |
             | podman       |
+
+        @microshift-preset
+        Examples:
```

Comment:
`Scenarios`

Reply:
I changed it to Scenarios and tested it that way:

```feature
Using existing bundle: /home/jsliacan/Downloads/crc_microshift_libvirt_4.13.6_amd64.crcbundle
Feature: Minimal user story
  User starts CRC cluster and checks status.

    Scenarios:
      | preset-value |
      | microshift   |
Deleted CRC instance (if one existed).

1 scenarios (1 passed)
4 steps (4 passed)
```
I'll keep `Examples` in the other Feature files for now. In `config.feature` it even makes sense that way.